vllm/sequence.py

```diff
@@ -588,6 +601,12 @@ def prompt_adapter_num_virtual_tokens(self) -> int:
         return self.prompt_adapter_request.prompt_adapter_num_virtual_tokens\
                          if self.prompt_adapter_request else 0
 
+    def init_multi_step(self, num_lookahead_slots: int) -> None:
```

Comment:
Why is the argument here `num_lookahead_slots`? My guess is that the name is from spec decode. I think the name is pretty confusing. Can we change it to something like `num_steps`?

Reply:
Similar comment: Can we combine `num_lookahead_slots` and `num_steps` in `SequenceGroupState`?

Reply:
combined in [22daa7e](https://github.com/vllm-project/vllm/pull/7000/commits/22daa7ed840a03f12ff89d933c7035176bd8ba1d)